network.js

```diff
@@ -15,6 +15,12 @@ export default {
       chainToViewConversionFactor: 1e-6,
       icon: `https://lunie.fra1.digitaloceanspaces.com/network-icons/cosmos.png`,
     },
+    {
+      viewDenom: 'slippers',
+      chainDenom: /^ibc*/g,
+      chainToViewConversionFactor: 1e-6,
+      icon: `https://lunie.fra1.digitaloceanspaces.com/network-icons/cosmos.png`,
+    },
```

Comment:
why? that doesn't look feasable. what about other denoms from IBC??

Reply:
we can have a reges as viewDenom, any string is good to go

Reply:
> what about other denoms from IBC??
please remove

Reply:
This is what I mean. You can have unlimited tokens. This is the problem I'm aiming to approach here

Reply:
you are not solving this problem. you are just calling them all `slippers`. ergo: please remove this non fix

Reply:
Yes, and I said replace `viewDenom` by another regex as the fix. Doing that